src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const pairs = Object.entries(robot);
+
+  for (const pair of pairs) {
+    const key = pair[0];
+    const val = pair[1];
+
+    newRobot[val] = key;
+  }
+
+  const keysRobotLength = Object.keys(robot).length;
+  const keysNewRobotLength = Object.keys(newRobot).length;
+
```

Comment:
interesting solution 🙂 
instead of comparing by length, in the loop, you can also check if the key is already in the object